packages/office-addin-dev-settings/src/sideload.ts

```diff
@@ -329,9 +338,8 @@ export async function sideloadAddIn(manifestPath: string, app?: OfficeApp, canPr
       await registerAddIn(manifestPath);
       // for Outlook, open Outlook.exe; for other Office apps, open the document
       if (app == OfficeApp.Outlook) {
-        const version = await getOutlookVersion();
-        if (version && semver.lt(version, "16.0.13709")) {
-          throw new ExpectedError(`The current version of Outlook (${version}) does not support sideload. Please use version 16.0.13709 or greater.`);
+        if (!(await hasOutlookVersion("16.0.13709"))) {
```

Comment:
It would be better to assign it to a variable first:

```
const version = await getOutlookVersion();

if (!hasOfficeVersion(...
```

I don't think really need the hasOutlookVersion() function.